components/shared/modals/ModalBody.vue

```diff
@@ -114,19 +115,13 @@ watch(
 </script>
 
 <style lang="scss">
-@import '@/assets/styles/abstracts/variables';
-
 $x-padding: 2rem;
 $t-padding: 1.5rem;
 $b-padding: 1.25rem;
 
 .modal-width {
   width: v-bind(modalWidth);
-  max-width: v-bind(modalMaxHeight);
-
-  @include mobile {
-    width: unset;
-  }
+  max-width: v-bind(modalMaxWidth);
```

Comment:
I don't think we need this anymore, any solution using tailwind?

Reply:
This?
``` vue
<div :class="`w-[unset] md:v-[${modalWidth}] md:max-w-[${modalMaxWidth}]`">
```

Reply:
yes this is better IMO